libsrc/Makefile

```diff
@@ -91,7 +93,7 @@ define INSTALL_recipe
 
 $(if $(prefix),,$(error variable `prefix' must be set))
 $(INSTALL) -d $(DESTDIR)$(datadir)/$(dir)
```

Comment:
The idea behind $(destdir) is to "prefix" the directory with a path that is not part of the installation itself. Any absolute path in the compiled executables, documentation or anything like that is *not* allowed to use $(destdir).

Look at it as follows: If you provide a $(destdir), and you chroot $(destdir) after the build, the installation should work. (Of course, you would need the required environment, too).

I need to use it in order to build the DEBian packages: The debian build environment sets $(destdir) to a temporary directory, installs the package there, and packs that directory afterwards. This way, you have a clean environment.

If $(destdir) does not work, you have to do much more steps. I know because I had to do it for some years for VICE, which did not support $(destdir) at that time. I believe I had fixed it before I left the team, though.